### PR TITLE
NetworkPlugin option in node config

### PR DESCRIPTION
**Pull request by rajatchopra**
@abhgupta review please

### CODE REVIEW

File: pkg/cmd/server/start/node_args.go [hunk 1]
```diff
@@ -42,6 +44,7 @@ func BindNodeArgs(args *NodeArgs, flags *pflag.FlagSet, prefix string) {
 	flags.StringVar(&args.VolumeDir, prefix+"volume-dir", "openshift.local.volumes", "The volume storage directory.")
 	// TODO rename this node-name and recommend hostname -f
 	flags.StringVar(&args.NodeName, prefix+"hostname", args.NodeName, "The hostname to identify this node with the master.")
+	flags.StringVar(&args.NetworkPluginName, prefix+"network-plugin", args.NetworkPluginName, "The network plugin to be called for configuring networking for pods.")
```
Comment (abhgupta) [inline pkg/cmd/server/start/node_args.go]: Do we really need to expose this argument here? The number of CLI args is something that we are generally trying to limit in favor of just having the configuration specified in the configuration file.

Comment (rajatchopra) [inline pkg/cmd/server/start/node_args.go]: Not stuck on it. Can definitely remove it. What if someone wants to specify it through the command line, though, just like volume-dir?
@smarterclayton ?

Comment (smarterclayton) [inline pkg/cmd/server/start/node_args.go]: Network plugin is probably an important flag.  How many other options would it have?

Comment (rajatchopra) [inline pkg/cmd/server/start/node_args.go]: None else that I foresee in the short term

**Comment by rajatchopra** [comment]
[merge]

**Pull request merged**